Content.Shared/Throwing/ThrownEvent.cs

```diff
@@ -0,0 +1,26 @@
+using JetBrains.Annotations;
+
+namespace Content.Shared.Throwing;
+
+/// <summary>
+///     Raised when throwing the entity in your hands.
+/// </summary>
+[PublicAPI]
+public sealed class ThrownEvent : HandledEntityEventArgs
```

Comment:
this should not parent handled event since:
- nothing actually handles it
- throwing system never checks ev.Handled